docs/fundamentals/networking/httpclient.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Make HTTP requests with the HttpClient
+description: Learn how to make HTTP requests and handle responses with the HttpClient in .NET.
+author: IEvangelist
+ms.author: dapine
+ms.date: 08/10/2022
+---
+
+# Make HTTP requests with the HttpClient
+
+Hypertext Transfer Protocol (or HTTP) is a protocol for requesting resources from a web server. The <xref:System.Net.Http.HttpClient?displayProperty=fullName> class exposes the ability to send HTTP requests and receive HTTP responses from a resource identified by a URI. There are many types of resources that are available on the web, and HTTP defines a set of request methods for accessing these resources. The request methods are differentiated via several factors, first by their _verb_ but also by the following characteristics:
+
+- A request method is **_idempotent_** if it can be successfully processed multiple times without changing the result. For more information, see [RFC 7231: Section 4.2.2 Idempotent Methods](https://datatracker.ietf.org/doc/html/rfc7231#section-4.2.2).
+- A request method is **_cacheable_** when its corresponding response can be stored for reuse. For more information, see [RFC 7231: Section 4.2.3 Cacheable Methods](https://datatracker.ietf.org/doc/html/rfc7231#section-4.2.3).
+- A request method is considered a **safe method** if it doesn't modify the state of a resource. All _safe methods_ are also _idempotent_, but not all _idempotent_ methods are considered _safe_. For more information, see [RFC 7231: Section 4.2.1 Safe Methods](https://datatracker.ietf.org/doc/html/rfc7231#section-4.2.1).
+
+| HTTP verb | Is idempotent | Is cacheable         | Is safe |
+|----------:|---------------|----------------------|---------|
```

Comment:
To me it looks funny to have the verbs right-aligned (here and in the other table) but perhaps you like it this way.